zerver/fixtures/bugdown-data.json

```diff
@@ -121,6 +121,12 @@
       "bugdown_matches_marked": true
     },
     {
+      "name": "escape",
+      "input": "\\**foo**",
+      "expected_output": "<p>**foo**</p>",
+      "bugdown_matches_marked": true
+    },
```

Comment:
Oh, maybe I now understand what you're saying -- the first `\` is a JSON escape and the second is escaping the actual `*` in the markdown input?  OK.